source/containers.tex

```diff
@@ -13630,9 +13630,9 @@
 \begin{itemdescr}
 \pnum
 \effects
-Equivalent to \tcode{c.append_range(std::forward<R>(rg))}
-if that is a valid expression,
-otherwise \tcode{ranges::copy(rg, back_inserter(c))}.
+Adds the elements of \tcode{rg} to \tcode{c}
```

Comment:
For other reviewers: `c.append_range(rg)` is equivalent to `c.append_range(std::forward<R>(rg))` since `append_range` for a standard container doesn't care about the value category of its `input_range` argument.

Dropping the "Equivalent to" Words of Power is non-editorial, however, which I failed to point out when we discussed this change earlier. "via" doesn't have the same fully-specified meaning. Could we rephrase this as `Equivalent to \tcode{c.append_range(rg)} if that is a valid expression, otherwise equivalent to \tcode{ranges::copy(rg, back_inserter(c))}.`?

Reply:
> Dropping the "Equivalent to" Words of Power is non-editorial

I was afraid of that. I do want to point out that the current wording is "Equivalent to ..." in 2 places and "Inserts ... via ..." in the 3rd place, which is again bizarrely inconsistent; and IIUC we can't harmonize all 3 in _either_ direction without one of the changes being non-editorial in this sense. I can bring this back in a real paper if we need me to.

Once I'm not trying to be editorial, I'll go even further and suggest replacing the word "via" with "as if by".